src/ui/public/agg_types/controls/time_interval.html

```diff
@@ -1,11 +1,26 @@
 <div class="form-group">
-  <label for="visEditorInterval{{agg.id}}">
-    Interval
+  <label
+    for="visEditorInterval{{agg.id}}"
+    i18n-id="common.ui.aggTypes.timeIntervalLabel"
+    i18n-default-message="Interval"
+  > 
     <icon-tip
+      ng-if="!(agg.buckets.getInterval().scale > 1)"
```

Comment:
Where is that coming from?

Reply:
It is coming from [<icon-tip content](https://github.com/elastic/kibana/pull/26091/files/140ba8b6adf7fe7b69cdfe0e118ffc64b0af2897#diff-9c4cc632606b4dfcdc13e6925104d4dfL8). I separate this icon  by two, with different default message according to condition (ng-if="agg.buckets.getInterval().scale > 1").